<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoNowPlayingWindow</class>
 <widget class="QMainWindow" name="VideoNowPlayingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Now Playing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="wmCloseButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>112</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
background: transparent;
border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>112</width>
      <height>56</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="toolbarOverlay" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>801</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="toolbarOverlay_1">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="buttonWidget" native="true">
       <layout class="QHBoxLayout" name="toolbarLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="shareButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentPositionLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="progressBar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="videoLenthLabel">
          <property name="text">
           <string>0:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="volumeLayout">
       <item>
        <widget class="QSlider" name="volumeSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="volumeButton">
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="controlOverlay" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="prevButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
background: transparent url(/etc/hildon/theme/mediaplayer/VideoOverlayButton.png);
border: none;
}

QToolButton:pressed
{
background: transparent url(/etc/hildon/theme/mediaplayer/VideoOverlayButtonPressed.png);
border: none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="playButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
background: transparent url(/etc/hildon/theme/mediaplayer/VideoOverlayButton.png);
border: none;
}

QToolButton:pressed
{
background: transparent url(/etc/hildon/theme/mediaplayer/VideoOverlayButtonPressed.png);
border: none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nextButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
background: transparent url(/etc/hildon/theme/mediaplayer/VideoOverlayButton.png);
border: none;
}

QToolButton:pressed
{
background: transparent url(/etc/hildon/theme/mediaplayer/VideoOverlayButtonPressed.png);
border: none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>wmCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoNowPlayingWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
